src/sheet.tsx

```diff
@@ -273,6 +274,9 @@ const Sheet = React.forwardRef<HTMLDivElement, SheetProps>(({onClose, children,
                                 aria-labelledby={modalTitleId}
                                 onScroll={onScroll}
                                 className={styles.children}
+                                style={{
```

Comment:
We can't use className because otherwise it won't detect the variable for acceptance tests